lms/djangoapps/instructor/views/api_urls.py

```diff
@@ -81,9 +81,6 @@
     # Cohort management
     url(r'^add_users_to_cohorts$', api.add_users_to_cohorts, name='add_users_to_cohorts'),
 
-    # Bulk enrollment management
-    url(r'enroll_users_to_course$', api.bulk_enroll_users_to_course, name='bulk_enroll_users_to_course'),
```

Comment:
Why we needed to remove the bulk enrollment feature? 
https://github.com/edly-io/edx-platform/pull/38/

Reply:
added back